sync/clients/sync.py

```diff
@@ -171,8 +196,6 @@ def onboard_workflow(self, workspace_id, job_id: str, project_id: str) -> dict:
                 "POST",
                 "/integrations/v1/databricks/workflow/onboard",
                 params={"workspace_id": workspace_id},
-                headers=headers,
-                content=content,
```

Comment:
Was this accidental?
https://api.synccomputing.com/docs#/Databricks%20Integration/onboard_workflow_integrations_v1_databricks_workflow_onboard_post

Reply:
Yeah this was just a stupid merge issue